conftest.py

```diff
@@ -210,32 +210,6 @@ def _load_module(module_name, path):
 _load_module("mscolab_settings", path)
 
 
-@pytest.fixture(autouse=True)
```

Comment:
removing this will Error all tests with a blocking Yes/No or Ok QMessageBox by a timeout